harness/tests/experiment/fixtures/lightning_adapter_onevar_model.py

```diff
@@ -17,23 +18,31 @@ def __getitem__(self, index: int) -> Tuple:
 
 
 class OneVarLM(pl.LightningModule):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, lr_frequency=1, *args, **kwargs):
+        super().__init__()
 
         model = torch.nn.Linear(1, 1, False)
         self.model = model
+        self.save_hyperparameters()
```

Comment:
Why do you need this line?

Reply:
to allow this line `                "frequency": self.hparams["lr_frequency"],` 
we could probably do it some other way too but this seems like the standard way of doing it in lightning adatper

Reply:
I see. `save_hyperparameters` always confuses me. But since it's part of the API, it's fine.